scripts/test_priority_plugin.sh

```diff
@@ -15,49 +16,55 @@ function stop_container() {
 	docker compose \
 		--file $DOCKER_COMPOSE_FILE \
 		--project-directory=$DOCKER_PROJECT_DIR \
+    --project-name="$COMPOSE_PROJECT_NAME" \
 		down
 }
 
 function start_container() {
 	docker compose \
 		--file $DOCKER_COMPOSE_FILE \
 		--project-directory=$DOCKER_PROJECT_DIR \
+		--project-name="$COMPOSE_PROJECT_NAME" \
 		up -d
 	# Copy slurm.conf to container
 	docker compose \
 		--file $DOCKER_COMPOSE_FILE \
 		--project-directory=$DOCKER_PROJECT_DIR \
+		--project-name="$COMPOSE_PROJECT_NAME" \
 		cp ./containers/docker-centos7-slurm/slurm.conf slurm:/etc/slurm/slurm.conf
 	# Copy priority plugin to container
 	docker compose \
 		--file $DOCKER_COMPOSE_FILE \
 		--project-directory=$DOCKER_PROJECT_DIR \
+		--project-name="$COMPOSE_PROJECT_NAME" \
 		cp \
 		./target/x86_64-unknown-linux-musl/debug/auditor-priority-plugin \
```

Comment:
If `RELEASE_MODE` is true, then the path changes to `./target/x86_64-unknown-linux-musl/release/auditor-priority-plugin`